src/gpuarray_buffer_cuda.c

```diff
@@ -190,7 +190,8 @@ static int setup_lib(void) {
       } while (res != GA_NO_ERROR && i < versions_length);
     }
     if (res != GA_NO_ERROR)
-      return res;
+      // Return the error from the original attempt
+      return load_libnvrtc(major, minor, global_err);
```

Comment:
This is not the original major/minor that you try again to generate the original error. But the guessed major/minor.

Reply:
That's true.

Reply:
Should be fixed now.